ci/docker/install/arm8_core.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
I'm confused, why do we need opencv on the host. We are cross compiling. don't you want opencv on the target instead?

https://airs.com/ian/configure/configure_5.html
https://landley.net/writing/docs/cross-compiling.html

Reply:
When I tried to get the Java API compiled on the Nano, it complained that I hadn't compiled MXNet with OpenCV. 
This error went away after I added OpenCV, but I have to say it added other issues to CI's deps.

Reply:
Considering that I needed to pull the Java API support because of cross-compile issues, I could unwind this OpenCV dep. However, one of my main use cases for the Nano is to plug in a camera, so I'm pretty sure an MXNet build with OpenCV is a good default for Nano devices.

Reply:
Is not a problem, I'm not saying you should unwind it. My question is that since it's installing in the host, I don't understand how it affects the target. Just a question not blocking the PR at all.